src/detail-panels/packs/packs.jsx

```diff
@@ -0,0 +1,112 @@
+// @ts-check
+
+import React, { useState } from 'react';
+import { useSearchParams } from 'react-router-dom';
+import { useResolveHandleOrDid } from '../../api';
+import { usePacksCreated, usePacksCreatedTotal, usePacksPopulated, usePacksPopulatedTotal } from '../../api/packs';
+import SearchIcon from '@mui/icons-material/Search';
+import { Button, CircularProgress } from '@mui/material'; 
+import { localise, localiseNumberSuffix } from '../../localisation';
+import { SearchHeaderDebounced } from '../history/search-header';
+import { useAccountResolver } from '../account-resolver';  
+import { PackView } from './pack-view';
+
+
+/**
+ * @param {Boolean | undefined} created 
+ */
+export function Packs(created=false){
```

Comment:
the type here is wrong. react component functions accept a single props argument which is an object. I think you intend for your props to include a single property named `created`

```suggestion
/**
 * @param {{ created: boolean }} props 
 */
export function Packs({ created }){
```